### PR TITLE
Gabriel/document

### PR DESCRIPTION
**Pull request by gabrielwry**
Document module first commit
working on test case (and potentially file path storage)

### CODE REVIEW

**Comment by jiashenC** [comment]
You should rebase you branch upon on `develop`. Same as another MR.

File: atlas/guarantor.py [hunk 1]
```diff
@@ -31,6 +32,7 @@ def wrapper(*args, **kwargs):
             try:
                 return func(*args, **kwargs)
             except all_exceptions as e:
+                print(str(e))
```
Comment (Arnoldosmium) [inline atlas/guarantor.py:35]: Remove this?

**Review by Arnoldosmium — COMMENTED** [review]
Some minor change

**Comment by Arnoldosmium** [comment]
My suggestion:

1. Merge in #10 
2. Merge in dev, Remove / change conflicts 
3. you can worry about test validity later (another PR). the most important parts are the `views` and `models` of the modules you worked on

**Pull request closed without merging**